### PR TITLE
[APP-588] fix inaccessible file path of video picker in > iOS 13

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/image_picker/ios/Classes/ImagePickerPlugin.m [hunk 1]
```diff
@@ -231,7 +231,7 @@ - (void)showPhotoLibrary {
 
 - (void)imagePickerController:(UIImagePickerController *)picker
     didFinishPickingMediaWithInfo:(NSDictionary<NSString *, id> *)info {
-  NSURL *videoURL = [info objectForKey:UIImagePickerControllerMediaURL];
+  NSString* videoURL = [[info objectForKey:UIImagePickerControllerMediaURL] path];
```
Comment (kahest) [inline packages/image_picker/ios/Classes/ImagePickerPlugin.m:234]: keep using NSURL - see below

File: packages/image_picker/ios/Classes/ImagePickerPlugin.m [hunk 2]
```diff
@@ -241,7 +241,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     return;
   }
   if (videoURL != nil) {
-    self.result(videoURL.path);
+    NSArray* spliteArray = [videoURL componentsSeparatedByString: @"/"];
+    NSString* lastString = [spliteArray lastObject];
```
Comment (kahest) [inline packages/image_picker/ios/Classes/ImagePickerPlugin.m:245]: use `videoURL.lastPathComponent`

**Comment by ghost** [comment]
Source: https://github.com/apache/cordova-plugin-camera/issues/506#issuecomment-534070130

File: packages/image_picker/ios/Classes/ImagePickerPlugin.m [hunk 3]
```diff
@@ -241,7 +241,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker
...
+    NSError *error;
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSString *documentsDirectory = [NSHomeDirectory() stringByAppendingPathComponent:@"tmp"];
```
Comment (kahest) [inline packages/image_picker/ios/Classes/ImagePickerPlugin.m:248]: use `NSTemporaryDirectory()`

**Pull request merged**